server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/HistogramAggregationBuilder.java

```diff
@@ -87,6 +91,7 @@ public static void registerAggregators(ValuesSourceRegistry.Builder builder) {
     private double offset = 0;
     private double minBound = Double.POSITIVE_INFINITY;
     private double maxBound = Double.NEGATIVE_INFINITY;
+    private DoubleBounds hardBounds;
```

Comment:
Should we convert `minBound` and `maxBound` into a `DoubleBounds` instance? if not, should probably just leave a comment that it's done this way for legacy reasons, not out of any specific need.

Reply:
I would like to refactor minBounds and maxBounds in a follow up PR. I just didn't feel like making this one more complicated than necessary. I will add TODO comment.